stl/inc/yvals_core.h

```diff
@@ -394,8 +394,9 @@
 // P2585R1 Improve Default Container Formatting
 // P2599R2 mdspan: index_type, size_type
 // P2604R0 mdspan: data_handle_type, data_handle(), exhaustive
-// P2613R1 mdspan: empty()
+// P2613R1 mdspan: empty()     

```

Comment:
There's extra whitespace. clang-format should trim that, or it can be done manually.